File: DecoratorDesignPattern/src/WithoutDecorator/MilkSugarCoffee.java
package WithoutDecorator;

public class MilkSugarCoffee implements Coffee {
    public String getDescription() {
        return "Simple Coffee + Milk + Sugar";
    }

    public double getCost() {
        return 5.0 + 1.5 + 0.5;
    }
}

File: design-patterns/python/Decorator-DesignPattern/src/without_decorator/milk_sugar_coffee.py
"""Milk and sugar coffee - separate class (no decorator)."""
from .coffee import Coffee


class MilkSugarCoffee(Coffee):
    def get_description(self) -> str:
        return "Simple Coffee + Milk + Sugar"

    def get_cost(self) -> float:
        return 5.0 + 1.5 + 0.5
